src/model_theory/basic.lean

```diff
@@ -47,19 +47,19 @@ namespace first_order
   type of relations of every natural-number arity. -/
 @[nolint check_univs] -- intended to be used with explicit universe parameters
 structure language :=
-(functions : ℕ → Type u) (relations : ℕ → Type v)
+(functions : ℕ → Type u) (relations : ℕ → Type u)
 
 namespace language
 
 /-- The empty language has no symbols. -/
-def empty : language := ⟨λ _, pempty, λ _, pempty⟩
+def empty_lang : language := ⟨λ _, pempty, λ _, pempty⟩
```

Comment:
This seems to violate the no-stuttering convention. How about making it `protected` instead, so you have to write `language.empty`? Or is there something else you want `language.empty` to mean?

Reply:
Protected is a better idea, thanks. I'd forgotten the syntax.